From cadd5f72c61e3fe974b20a44316feb22d31f0533 Mon Sep 17 00:00:00 2001
From: Dominique Leuenberger <dimstar@opensuse.org>
Date: Thu, 15 Nov 2012 14:58:50 +0100
Subject: [PATCH] include config.h

As per binutils, we are FORCED to include config.h when we want
to make use of bfd.h

See http://sourceware.org/bugzilla/show_bug.cgi?id=14243
---
 src/memprof.h |    1 +
 1 file changed, 1 insertion(+)

diff --git a/src/memprof.h b/src/memprof.h
index 9d04a1a..9fc7cf0 100644
--- a/src/memprof.h
+++ b/src/memprof.h
@@ -20,6 +20,7 @@
  */
 /*====*/
 
+#include "config.h"
 #include <glib.h>
 #include <gtk/gtkwidget.h>
 #include <stdio.h>
-- 
1.7.10.4

